docs/publishers/config.rst

```diff
@@ -450,6 +450,39 @@ loads.
                               // relative from the receiving iframe.
     }
 
+.. option:: sideBySide
+
+  ``Object``. This option lets you customize how the side-by-side mode behaves.
+
+  .. note::
+
+    The side-by-side mode allows the space used by the web page's main content
+    area to be adapted while the sidebar is open, ensuring it does not overlap
+    with potentially annotatable content.
+
+  For example:
+
+  .. code-block:: javascript
+
+     window.hypothesisConfig = () => ({
+       sideBySide: {
+         mode: 'manual'
+       }
+     });
+
+  The following keys are supported in the :option:`sideBySide` object.
+
+  .. option:: mode
+
+    ``auto`` or ``manual``. Auto is the default value, where the main page
+    content will be adjusted based on the sidebar status, while manual
+    indicates the web page wants to take full control of handling side-by-side,
+    which will effectively disable that logic.
+
+    When setting it to ``manual``, you probably want to
+    `listen for layout changes </publishers/events/#cmdoption-arg-hypothesis-layoutchange>`_
+    as well, so that you can adapt the main content area of the page.
```

Comment:
I think we should re-word this to indicate that web page authors _should_ respond to layout changes and adapt their content if that is reasonably possible, so as to provide an experience that is consistent with how Hypothesis works more generally. There will be cases where it is not reasonable, in which case the page author can just set the mode to "manual" and do nothing else.

Possible wording: "When setting this to `manual`, you should listen for layout changes and adapt content to fit alongside the sidebar, if it is reasonable to do so."